macchanger: edit page (#12459)
tldr-pages/tldr: pages/linux/macchanger.md

`macchanger --random {{interface}}`

> - Set interface to a random MAC, and pretend to be a [b]urned-[i]n-[a]ddress:

Reviewer: ```suggestion
- Set an interface to a random MAC address, and pretend to be a [b]urned-[i]n-[a]ddress:
```

`macchanger --random --bia {{interface}}`

> - Set interface to a random vendor MAC of same/any kind:

Reviewer: ```suggestion
- Set an interface to a random vendor MAC address of same/any kind:
```

Also, please clarify: same kind with regards to what?

Author: I will remove this example. I have tested it, and it the tool does not comply with the documentation. So this is a buggy feature.

`macchanger -{{a|A}} {{interface}}`

> - Set interface to a specific MAC:

Reviewer: ```suggestion
- Set interface to a specific MAC address:
```

`macchanger --mac {{XX:XX:XX:XX:XX:XX}} {{interface}}`

> - Print identification (first 3 pairs of MAC) of all known vendors:

Reviewer: ```suggestion
- Print the identifications (the first three bytes of a MAC address) of all known vendors:
```

`macchanger --list`

> - Reset interface to its permanent hardware MAC:

Reviewer: ```suggestion
- Reset an interface to its permanent hardware MAC address:
```
